doc/lib/nf3.mjs.html

```diff
@@ -727,7 +727,7 @@ <h1 class="page-title">Source: nf3.mjs</h1>
   value of each propery is the number of tokens originating from that contract.
   */
   async getLayer2Balances() {
-    const res = await axios.get(`${this.clientBaseUrl}/commitment/balance`);
+    const res = await axios.post(`${this.clientBaseUrl}/commitment/balance`);
```

Comment:
We are requesting data through this API call. It should be `post`

Reply:
done

Reply:
It was my mistake. I meant `get`. Apologies

Reply:
This hasn't been changed. But accepting for now. Please change in a future PR to not have further delay for this merge